src/lib.rs

```diff
@@ -133,13 +135,35 @@ pub fn build(
                 enable_coverage,
                 extra_args,
             )?;
+        } else {
+            anyhow::bail!("Unsupported target OS for x86_64");
         }
     } else if cfg!(target_arch = "aarch64") {
-        if cfg!(target_os = "macos") {
+        if cfg!(target_os = "linux") {
+            if cfg!(target_env = "gnu") {
+                platforms::aarch64_linux_gnu::build(
+                    build_type,
+                    enable_tests,
+                    enable_coverage,
+                    extra_args,
+                )?;
+            } else if cfg!(target_env = "musl") {
+                platforms::aarch64_linux_musl::build(
```

Comment:
Can we test this as well in your workflow?

Reply:
i mean it already builds successfully against these platforms